packages/cli/src/index.ts

```diff
@@ -18,6 +18,7 @@ program
   .command('dev')
   .description('Start a Novu Dev Studio and a localtunnel')
   .option('-p, --port <value>', 'Set the local port for the Novu endpoint, defaults to 4000', '4000')
+  .option('--path <value>', 'Set the Novu Endpoint mounted path, defaults to 4000', '/api/novu')
```

Comment:
Can users override that now in @novu/framework?

Reply:
This is not really a novu/framework concern, as the user can mount the endpoint anywhere he really wants. We just need to make that the Local Studio knows which path to access

![CleanShot 2024-06-27 at 09 10 26@2x](https://github.com/novuhq/novu/assets/8872447/c10950a6-222c-4409-9ffc-55cf306177f7)